README.md

```diff
@@ -127,7 +127,7 @@ To use this option, uncomment the line and change the location of `"path/to/fold
 existing directory on your laptop.
 
 By the time you read this hint, your VM is most likely already up and running. Not to worry: just
-uncomment the above mentioned line in your `Vagrantfile` and run `vagrant reload`. This will restart
+uncomment the above mentioned line in your `Vagrantfile` and run `vagrant reload --no-provision`. This will restart
```

Comment:
Why do we need --no-provision, it is there by default. I tested it again. Do you see different behaviour?